components/forms/Form/Form.tsx

```diff
@@ -151,70 +151,90 @@ const InnerForm: React.FC<InnerFormProps> = (props) => {
        * otherwise GET request will be sent which will result in leaking all the user data
        * to the URL
        */}
-      <form
-        id="form"
-        data-testid="form"
-        method="POST"
-        onSubmit={(e) => {
-          e.preventDefault();
-          if (timerActive) {
-            if (!formTimerState.canSubmit) {
-              window.dataLayer = window.dataLayer || [];
-              window.dataLayer.push({
-                event: "form_submission_spam_trigger",
-                formID: formID,
-                formTitle: form.titleEn,
-                submitTime: formTimerState.remainingTime,
-              });
-              setSubmitTooEarly(true);
-              // In case the useEffect timer failed check again
-              //formTimerDispatch({ type: "check" });
-              return;
-            }
-            // Only change state if submitTooEarly is already set to true
-            submitTooEarly && setSubmitTooEarly(false);
-          }
+      {
+        <>
+          <RichText>
+            {form.introduction &&
+              form.introduction[props.language == "en" ? "DescrptionEn" : "DescriptionFr"]}
```

Comment:
Typo on the en + should be descriptionEn or ``descriptionFr`

Reply:
Given this is public facing for some versions thinking should also check ton ensure the description exists